readme.md

```diff
@@ -5,10 +5,25 @@
 [![License](http://img.shields.io/cocoapods/l/SupportKit.svg)](http://supportkit.io/terms.html)
 
 # tl;dr
-    Sign up and get an app token: https://app.supportkit.io/signup
+
+
+Sign up and get an app token: [From this link](https://app.supportkit.io/signup).
```

Comment:
Thinking of [a post](http://www.smashingmagazine.com/2012/06/20/links-should-never-say-click-here/) I read recently on "Click Here" type links. Perhaps the words "Sign up" should be made clickable.

    [Sign up](https://app.supportkit.io/signup) and get an app token.

OR

    [Sign up](https://app.supportkit.io/signup) with SupportKit and get an app token.